x-pack/legacy/plugins/apm/cypress/tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
   "extends": "../../../../tsconfig.json",
-  "exclude": [],
+  "exclude": ["./node_modules/**/*"],
   "include": ["./**/*"],
   "compilerOptions": {
-    "types": ["cypress", "node"]
+    /* Basic Options */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    /* Strict Type-Checking Options */
+    "strict": true,                           /* Enable all strict type-checking options. */
+    "esModuleInterop": true                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
```

Comment:
This is coming from the [examples](https://github.com/cypress-io/cypress-example-recipes/blob/a240054d7f5626ffcd7bd668dded96d219c4a7eb/examples/preprocessors__typescript-webpack/tsconfig.json). Please take a look 🙏

Reply:
I think we can just revert this file and keep it as it was